Applications/AdminCli/src/AdminCli/Properties/launchSettings.json

```diff
@@ -1,21 +1,22 @@
 {
-    "profiles": {
-        "Start": {
-            "commandName": "Project",
-            "commandLineArgs": "tier list",
-            // enter an existing command here in order to debug it
-            //"commandLineArgs": "client list", // enter an existing command here in order to debug it
-            //"commandLineArgs": "client create --clientId test --clientSecret test --defaultTier Basic --maxIdentities 100", // enter an existing command here in order to debug it
-            //"commandLineArgs": "client delete test", // enter an existing command here in order to debug it
-            //"commandLineArgs": "token reset-access-failed-count --id TOKJyKNScncbXqqMEsGf",
-            // enter an existing command here in order to debug it
-            //"commandLineArgs": "announcement send --expiration 2025-12-31 --severity medium",
-            "environmentVariables": {
-                "Database__Provider": "Postgres",
-                //"Database__ConnectionString": "Server=localhost;Database=enmeshed;User Id=devices;Password=Passw0rd;TrustServerCertificate=True;" //SqlServer
-                "Database__ConnectionString": "User ID=postgres;Password=admin;Server=localhost;Port=5432;Database=enmeshed;"
-                //Postgres
-            }
-        }
+  "profiles": {
+    "Start": {
+      "commandName": "Project",
+      // enter an existing command here in order to debug it
+      //"commandLineArgs": "announcement list",
+      "commandLineArgs": "announcement delete --id ANCXYZ2YKRnc9SLnEmMy",
+      //"commandLineArgs": "client list", // enter an existing command here in order to debug it
+      //"commandLineArgs": "client create --clientId test --clientSecret test --defaultTier Basic --maxIdentities 100", // enter an existing command here in order to debug it
+      //"commandLineArgs": "client delete test", // enter an existing command here in order to debug it
+      //"commandLineArgs": "token reset-access-failed-count --id TOKJyKNScncbXqqMEsGf",
+      // enter an existing command here in order to debug it
+      //"commandLineArgs": "announcement send --expiration 2025-12-31 --severity medium",
+      "environmentVariables": {
+        "Database__Provider": "Postgres",
+        //"Database__ConnectionString": "Server=localhost;Database=enmeshed;User Id=devices;Password=Passw0rd;TrustServerCertificate=True;" //SqlServer
+        "Database__ConnectionString": "User ID=postgres;Password=admin;Server=localhost;Port=5432;Database=enmeshed;"
+        //Postgres
+      }
```

Comment:
Could you delete the whole `environmentVariables` property please? Just noticed that I forgot to do that during a big refactoring of the Admin CLI I did about 2 weeks ago.